doc/source/installation.rst

```diff
@@ -1,24 +1,117 @@
+
+_______________________
+
+.. _backend-drivers:
+
+Backends drivers
+----------------
+
+As mentioned above, we provide backends for quantum simulation on classical
+hardware and quantum hardware management and control. In the image below we
+present a schematic view of the currently supported backends.
+
+.. image:: backends.png
+
+Quantum simulation is proposed through dedicated backends for single node
+multi-GPU and multi-threading CPU setups. Quantum hardware control is under
+development.
+
+_______________________
+
+Packages
+--------
+
+Following the overview description above, in this section we present the python
+packages for the modules and backends presented.
+
+Base package
+^^^^^^^^^^^^
+
+* :ref:`installing-qibo` is the base package for coding and using the API. This package contains all primitives and algorithms for start coding quantum circuits, adiabatic evolution and more (see :ref:`Components`). This package comes with a lightweight quantum simulator which works on multiple CPU architectures such as x86 and arm64.
+
+.. _simulation-backends:
+
+Simulation backends
+^^^^^^^^^^^^^^^^^^^
+
+We provide multiple simulation backends for Qibo, which are automatically loaded if the corresponding packages are installed, following the hierarchy below:
+
+* :ref:`installing-qibotf` (*recommended*): an efficient simulation backend for CPU, GPU and multi-GPU based on TensorFlow primitives. Install this package if you need to simulate quantum circuits with large number of qubits or complex quantum algorithms which may benefit from computing parallelism.
+* :ref:`installing-tensorflow`: a pure TensorFlow implementation for quantum simulation which provides access to gradient descent optimization and the possibility to implement classical and quantum architectures together. This backend is not optimized for memory and speed, use :ref:`installing-qibotf` instead.
+* :ref:`installing-numpy`: a lightweight quantum simulator shipped with the :ref:`installing-qibo` base package. Use this simulator if your CPU architecture is not supported by the other backends.
 
 _______________________
 
-.. _installing-with-pip:
+Operating systems support
+-------------------------
+
+In the table below we summarize the status of *pre-compiled binaries
+distributed with pypi* for the packages listed above.
+
++------------------+------+------------------+------------+
+| Operating System | qibo | qibotf (cpu/gpu) | tensorflow |
++==================+======+==================+============+
+| Linux x86        | Yes  | Yes/Yes          | Yes        |
++------------------+------+------------------+------------+
+| MacOS >= 10.15   | Yes  | Yes/No           | Yes        |
++------------------+------+------------------+------------+
+| Windows          | Yes  | No/No            | No         |
++------------------+------+------------------+------------+
+
+.. note::
+      All packages are supported for Python 3.6, 3.7 and 3.8.
```

Comment:
Just a reminder to include 3.9 here once we make the relevant qibotf upgrade and include 3.9 in qibo CI.